pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/KitRequestShipping.java

```diff
@@ -84,24 +84,8 @@
 @JsonInclude(JsonInclude.Include.NON_NULL)
 public class KitRequestShipping extends KitRequest implements HasDdpInstanceId {
 
-    public static final String SQL_SELECT_KIT_REQUEST_NEW =
```

Comment:
I think this was used for selecting new eligible kits for ordering in gbf for testboston, but we got rid of most of that code long ago